examples/angular-cli/src/stories/addon-knobs.stories.ts

```diff
@@ -20,11 +20,23 @@ storiesOf('Addon Knobs', module)
   .addDecorator(withKnobs)
   .add('Simple', () => {
     const name = text('name', 'John Doe');
-    const age = number('age', 44);
+    const age = number('age', 0);
     const phoneNumber = text('phoneNumber', '555-55-55');
 
     return {
-      component: SimpleKnobsComponent,
+      moduleMetadata: {
+        entryComponents: [SimpleKnobsComponent],
+        declarations: [SimpleKnobsComponent],
+      },
+      template: `
+        <h1> This is a template </h1>
```

Comment:
You need now update the snapshot 😅

Reply:
I tried but got this error :( 
```
yarn run v1.3.2
$ node ./scripts/test.js
? Select which tests to run Update all snapshots (update)
● Validation Error:

  Module <rootDir>/node_modules/react-native/jest/assetFileTransformer.js in the transform option was not found.

  Configuration Documentation:
  https://facebook.github.io/jest/docs/configuration.html
```

Reply:
try `node node_modules\jest\bin\jest.js angulatshots -u`

Reply:
`node node_modules/jest/bin/jest.js examples/angular-cli/angularshots.test.js -u` did the trick :)

Reply:
![image](https://user-images.githubusercontent.com/7867954/35064213-f98cfe5a-fbd1-11e7-8423-a1f700218298.png)